packages/ui/src/components/primitives/Message.tsx

```diff
@@ -60,99 +135,216 @@ export const MessageAuthorArea = () => {
 	);
 };
 
-export const MessageContents = () => {
+const LongMessageContents = () => {
```

Comment:
Get rid of expand / collapse of message content

Reply:
Done :+1: